art/metrics/privacy/worst_case_mia_score.py

```diff
@@ -0,0 +1,122 @@
+# MIT License
+#
+# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2020
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""
+This module implements metric for inference attack worst case accuracy measurement.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+from typing import Optional, List, Tuple, Union
+
+import logging
+import numpy as np
+from sklearn.metrics import roc_curve
+
+
+TPR = float
+FPR = float
+THR = float
+
+
+def _calculate_roc_for_fpr(y_true, y_proba, targeted_fpr):
+    """Get FPR, TPR and, THRESHOLD based on the targeted_fpr (such that FPR <= targeted_fpr)"""
+    fpr, tpr, thr = roc_curve(y_true=y_true, y_score=y_proba)
+    # take the highest fpr and an appropriated threshold that achieve at least FPR=fpr
+    if np.isnan(fpr).all() or np.isnan(tpr).all():
+        logging.warning("TPR or FPR values are NaN")
+        return None, None, None
+
+    targeted_fpr_idx = np.where(fpr <= targeted_fpr)[0][-1]
+    return fpr[targeted_fpr_idx], tpr[targeted_fpr_idx], thr[targeted_fpr_idx]
+
+
+def get_roc_for_fpr(  # pylint: disable=C0103
```

Comment:
Why has C0103 been disabled?

Reply:
I will remove and rerun the pylint

Reply:
addressed in 5f10754f0a68b9f0b4a476fc6b882e2938684d30